core/meta/inc/TVirtualStreamerInfo.h

```diff
@@ -180,6 +180,14 @@ class TVirtualStreamerInfo : public TNamed {
    static void         SetCanDelete(Bool_t opt=kTRUE);
    static void         SetFactory(TVirtualStreamerInfo *factory);
 
+   // \brief Generate the TClass and TStreamerInfo for the requested pair.
+   // This creates a TVirtualStreamerInfo for the pair and trigger the BuildCheck/Old to
+   // provokes the creation of the corresponding TClass.  This relies on the dictionary for
+   // std::pair<const int, int> to already exist (or the interpreter information being available)
+   // as it is used as a template.
```

Comment:
```suggestion
   // provokes the creation of the corresponding TClass.
```
Implementation detail; this should be asserted by `GenerateInfoForPair`, not be part of the doc.